resources/views/v4/admin/request_key.blade.php

```diff
@@ -0,0 +1,75 @@
+@extends('v4.layouts.app')
```

Comment:
wait.... the page to request a key is not an admin function.... the developer needs to access this page (without any authentication challenge)